test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegateMaterialX.py

```diff
@@ -71,8 +69,15 @@ def _StartTest(self, testName):
     def testUVStreamManagement(self):
         """Test that a scene without primvar readers renders correctly if it
            uses indexed UV streams"""
+        cmds.file(force=True, new=True)
+        cmds.move(2, -2, 1.5, 'persp')
         self._StartTest('MtlxUVStreamTest')
 
+    def testMayaSurfaces(self):
+        cmds.file(force=True, new=True)
+        cmds.move(6, -6, 6, 'persp')
+        cmds.rotate(60, 0, 45, 'persp')
+        self._StartTest('MayaSurfaces')
```

Comment:
File includes a poly cube with standard surface MaterialX shading to show that the faces with Z-pointing normals render correctly now.